tests/test_ssl.py

```diff
@@ -3129,24 +3240,26 @@ def test_total_renegotiations(self):
         connection = Connection(Context(TLSv1_METHOD), None)
         self.assertEquals(connection.total_renegotiations(), 0)
 
-#     def test_renegotiate(self):
-#         """
-#         """
-#         server, client = self._loopback()
+    def test_renegotiate(self):
```

Comment:
Should this test call `renegotiate_pending` during and after to confirm it properly returns True/False at the expected time?